README.md

```diff
@@ -82,6 +82,7 @@ python setup.py install
 
 # Or (for dev mode)
 pip install -e .[dev]
+pre-commit install
```

Comment:
I hv no idea about this. is it related to installation?

Reply:
For dev mode, if `pre-commit` is not installed in the repo, the checks won't work like `yapf`, `flake8`.